docker-compose.yml

```diff
@@ -1,10 +1,12 @@
-version: '3.1'
+version: '2'
```

Comment:
We update these to stable Docker version (not `.0` releases, anymore) as they come out.